.ci/test_r_package_windows.ps1

```diff
@@ -39,12 +39,24 @@ function Remove-From-Path {
 # remove some details that exist in the GitHub Actions images which might
 # cause conflicts with R and other components installed by this script
 $env:RTOOLS40_HOME = ""
+Remove-From-Path ".*Amazon.*"
+Remove-From-Path ".*Anaconda.*"
+Remove-From-Path ".*android.*"
+Remove-From-Path ".*Android.*"
 Remove-From-Path ".*chocolatey.*"
 Remove-From-Path ".*Chocolatey.*"
-Remove-From-Path ".*Git.*mingw64.*"
+Remove-From-Path ".*\\Git\\.*"
+Remove-From-Path ".*hostedtoolcache.*"
```

Comment:
Strange folder with preinstalled software. I think they can install more in that folder and we'll be prepared.

![image](https://user-images.githubusercontent.com/25141164/125173108-02b32080-e1c6-11eb-9d35-3d79d8298c19.png)